services/gateway/sources/version3/mappings/blockchainApp.js

```diff
@@ -16,7 +16,7 @@
 module.exports = {
 	name: '=,string',
 	chainID: '=,string',
-	state: '=,string',
+	status: '=,string',
```

Comment:
With this change, we also need to update integration test schema at `tests/schemas/api_v3/blockchainAppsSchema.schema.js`

Reply:
Applied.